.github/workflows/test.yaml

```diff
@@ -58,7 +58,7 @@ jobs:
         id: html-artifact
         env:
           COMMIT_SHA: ${{ github.event.pull_request.head.sha || github.sha }}
-        if: ${{ !cancelled() && (steps.changed-files.outputs.any_changed == 'true' && github.event_name == 'pull_request')}}
+        if: ${{ !cancelled() && (steps.changed-files.outputs.any_changed == 'true' && github.event_name == 'pull_request') }}
```

Comment:
Unrelated diff? +1 on the change, let's do it in a separate PR?

Reply:
#1250